index.js

```diff
@@ -67,18 +76,25 @@ module.exports = {
     'jsx-a11y/role-supports-aria-props': 1,
     'jsx-a11y/tabindex-no-positive': 1,
     'linebreak-style': 2,
-    'max-len': [2, 120, 2],
+    'max-len': [
+      2,
+      {
+        code: 120,
+        tabWidth: 2,
+        ignoreStrings: true,
```

Comment:
`prettier` will not autoformat lines that go over the linelength if they are like, long strings. so for example,
```
const ProgramDetailsSettingsContainer = require('bundles/admin-dashboard-programs/components/ProgramDetailsSettingsContainer');
```
which previously got fixed to
```
const ProgramDetailsSettingsContainer = 
  require('bundles/admin-dashboard-programs/components/ProgramDetailsSettingsContainer');
```
will just stay on the same line. this rule just makes sure eslint doesn't warn about this.